src/test/java/jpa/BaseEntityTest.java

```diff
@@ -0,0 +1,31 @@
+package jpa;
+
+import jpa.line.Line;
+import jpa.line.LineRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@SpringBootTest
+public class BaseEntityTest {
+
+    @Autowired
+    private LineRepository lineRepository;
+
+    @Test
+    void Auditing_테스트() {
+        LocalDateTime now = LocalDateTime.of(2020, 12, 28, 14, 48);
```

Comment:
```suggestion
        LocalDateTime now = LocalDateTime.now();
```
테스트 잘 작성해주셨어요 ㅎㅎ 그냥 now() 써도 될 것 같네요 :)